package.json

```diff
@@ -11,8 +11,8 @@
   },
   "devDependencies": {
     "admin-lte": "2.3.11",
-    "axios": "^0.16.2",
-    "bootstrap": "^3.3.7",
+    "axios": "^0.18",
+    "bootstrap": "^4.0.0",
```

Comment:
Ooh someone is brave... are we sure this is going to be as simple as a upgrade?

Reply:
Tested and appears to be working fine (Laravel now ships with 4.0.0 out of the box). Feel free to have a test!

Reply:
+1 with the bootstrap upgrade, that will break quite a lot. Off the top of my head, panels are now cards to start with